src/igor/extend.go

```diff
@@ -57,10 +57,13 @@ func runExtend(cmd *Command, args []string) {
 	}
 
 	user, err := getUser()
+
```

Comment:
@jcrussell : I'm not trying to nit-pick, but is it typical to have an empty line between a call and checking the err from the call?

Reply:
we added that line so we can test my no-ouo settings. do you want me to repush?

Reply:
I don't put a newline.

Reply:
changed

Reply:
@csymonds this one.